README.md

```diff
@@ -21,7 +21,22 @@ Download the installer from https://go.dev/dl/ and follow the installation instr
 ```
 
 ### 2. Install Rust
-Make sure you have Rust installed by following the instructions [here](https://rustup.rs/)..
+Make sure you have Rust installed by following the instructions [here](https://rustup.rs/).
+
+## Logging Configuration
+
+The verifier uses two logging systems:
+
+- Rust logging via `tokio-tracing`, configured through the `RUST_LOG` environment variable. See the [tracing documentation](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html) for detailed configuration options.
+- Go logging for FFI calls, configured through the `SP1_GO_LOG` environment variable.
+
+To set a global log level (e.g. warn), configure both variables:
+
+```bash
+RUST_LOG=warn SP1_GO_LOG=warn ./plonk-verify ...
+```
+
+Valid log levels are: error, warn, info, debug, trace
```

Comment:
Since both `RUST_LOG` and `SP1_GO_LOG` support `none` for disabling the logging, and the go code defaults to `info` we should mention `off` here:
```suggestion
Valid log levels are: error, warn, info, debug, trace, off
```